README.md

```diff
@@ -9,43 +9,74 @@ The process for submitting an RFC is documented in the [RFC Process](CONTRIBUTIN
 
-  A basic Application Layer protocol that uses HTTPS to exchange details needed to set up an Interledger payment.
 
-* **[10: Connector-To-Connector Protocol (CCP)](0010-connector-to-connector-protocol/0010-connector-to-connector-protocol.md)**
+* **[3: Interledger Protocol (ILP)](0003-interledger-protocol/0003-interledger-protocol.md)**
 
-  The protocol used by connectors to communicate routing information.
+  Specifies the Interledger Protocol and Interledger Packet, which are used for sending payment instructions across different ledgers and connectors. This is the core protocol in the Interledger stack.
 
-* **[11: Interledger Payment Request (IPR)](0011-interledger-payment-request/0011-interledger-payment-request.md)**
+* **[8: Interledger Quoting Protocol (ILQP)](0008-interledger-quoting-protocol/0008-interledger-quoting-protocol.md)**
 
-  A Transport Layer protocol in which the receiver generates the ILP Packet and condition.
+  Specifies the protocol used for requesting a quote for an Interledger payment from a connector.
 
 * **[15: ILP Addresses](0015-ilp-addresses/0015-ilp-addresses.md)**
 
   Specifies the Interledger Address format for ledgers and accounts.
 
+## Protocols Built Upon ILP
+
+* **[9: Simple Payment Setup Protocol (SPSP)](0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md)**
+
+  A basic Application Layer protocol that uses HTTPS to exchange details needed to set up an Interledger payment.
+
+* **[11: Interledger Payment Request (IPR)](0011-interledger-payment-request/0011-interledger-payment-request.md)**
+
+  A Transport Layer protocol in which the receiver generates the ILP Packet and condition.
+
 * **[16: Pre-Shared Key (PSK)](0016-pre-shared-key/0016-pre-shared-key.md)**
 
   The recommended Transport Layer protocol for most use cases, which uses a secret shared between the sender and receiver to generate the condition, authenticate the packet, and encrypt application data.
 
+## Ledger Layer
+
+* **[4: Ledger Plugin Interface](0004-ledger-plugin-interface/0004-ledger-plugin-interface.md)**
+
+  Ledger abstraction used in the JavaScript implementation. This can be used as a model for defining such plugins in other languages.
+
+* **[12: Five Bells Ledger API](0012-five-bells-ledger-api/0012-five-bells-ledger-api.md)**
+
+  Specifies an API that centralized ledgers may implement to provide clients and connectors with all functionality needed for Interledger.
+
+* **[17: Ledger Requirements](0017-ledger-requirements/0017-ledger-requirements.md)**
+
+  Outlines the degrees of Interledger support that different ledgers may provide.
+
+* **[21: Plugin RPC API](0021-plugin-rpc-api/0021-plugin-rpc-api.md)**
+
+  Recommended API for trustlines and payment channels.
+
+## Routing
+
+* **[10: Connector-To-Connector Protocol (CCP)](0010-connector-to-connector-protocol/0010-connector-to-connector-protocol.md)**
+
+  The protocol used by connectors to communicate routing information.
```

Comment:
maybe put 'A protocol' instead of 'The protocol', because it's not being used everywhere